WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/CustomAmounts/AddCustomAmountViewModelTests.swift

```diff
@@ -131,4 +118,34 @@ final class AddCustomAmountViewModelTests: XCTestCase {
         // Then
         XCTAssertNotNil(analytics.receivedEvents.first(where: { $0 == WooAnalyticsStat.addCustomAmountNameAdded.rawValue }))
     }
+
+    func test_showPercentageInput_when_baseAmountForPercentage_is_zero_then_returns_false() {
+        // When
+        let viewModel = AddCustomAmountViewModel(baseAmountForPercentage: 0, onCustomAmountEntered: {_, _, _ in })
+
+        // Then
+        XCTAssertFalse(viewModel.showPercentageInput)
+    }
+
+    func test_showPercentageInput_when_baseAmountForPercentage_is_bigger_than_zero_then_returns_true() {
+        // When
+        let viewModel = AddCustomAmountViewModel(baseAmountForPercentage: 1, onCustomAmountEntered: {_, _, _ in })
+
+        // Then
+        XCTAssertTrue(viewModel.showPercentageInput)
+    }
+
```

Comment:
Just for completion and to assure that we're not breaking something in the future we can add an extra test checking for a negative `baseAmountForPercentage`:

```
    func test_showPercentageInput_when_baseAmountForPercentage_is_negative_then_returns_false() {
        // When
        let viewModel = AddCustomAmountViewModel(baseAmountForPercentage: -1, onCustomAmountEntered: {_, _, _ in })

        // Then
        XCTAssertFalse(viewModel.showPercentageInput)
    }
```